Mage.Sets/src/mage/cards/i/IllithidHarvester.java

```diff
@@ -106,7 +105,7 @@ public boolean apply(Game game, Ability source) {
         Predicate<Permanent> pred = new PermanentIdPredicate(UUID.randomUUID());
         for (Target target : source.getTargets()) {
             for (UUID targetId : target.getTargets()) {
-                if (!game.getPermanent(targetId).isTransformable()) {
+                if (!game.getPermanent(targetId).isTransformable() && !(game.getCard(targetId) instanceof ModalDoubleFacedCardHalf)) {
```

Comment:
Looks like that code it can be extracted to useful MageObject::canFaceDown method and used on other face down related places.

Reply:
Done. Also included a check for if the card is already face down which was previously done inconsistently